README.md

```diff
@@ -127,9 +127,20 @@ your desired [configuration](https://opentelemetry.io/docs/collector/configurati
 from it in the namespace you created above:
 
 ```
-kubectl create configmap otel-config --from-file=./otel-config.yaml -n $OTEL_NAMESPACE
+make k8s-config
```

Comment:
I like the idea of a user being able to run `make`, and have everything be installed in one command.  But I feel like part of the point of this is educational: show users the full command so they understand what the steps are.  For this, I would lean more towards showing the full command.

Reply:
Yeah, I was on the fence about this. I'm not sure if our audience is comfortable or not with longer kubectl commands like this. @jsuereth wdyt?